relay-server/src/services/processor.rs

```diff
@@ -2015,16 +2015,18 @@ impl EnvelopeProcessorService {
     async fn process_sessions(
         &self,
         managed_envelope: &mut TypedEnvelope<SessionGroup>,
+        config: Arc<Config>,
         project_info: Arc<ProjectInfo>,
         rate_limits: Arc<RateLimits>,
     ) -> Result<Option<ProcessingExtractedMetrics>, ProcessingError> {
         let mut extracted_metrics = ProcessingExtractedMetrics::new();
 
         session::process(
             managed_envelope,
+            &self.inner.global_config.current(),
+            &config,
```

Comment:
Here, we are getting passed the `config` from outside, but reading the `global_config` from `self.inner`. I did it this way because that's how it is in other processors (e.g. replays), but I don't know whether there's a good reason for it or it's a historical accident.